proc/proc_windows.go

```diff
@@ -127,6 +127,20 @@ func Launch(cmd []string) (*Process, error) {
 		return nil, ProcessExitedError{Pid: dbp.Pid, Status: exitCode}
 	}
 
+	for _, thread := range dbp.Threads {
```

Comment:
It seems like maybe that is unnecessary (the first `WaitForDebugEvent`). I assume it wouldn't trigger an event as the process is simply created, but never executed any instructions. This is distinct from how we handle process creation on UNIX systems where we wait for that first `PTRACE_TRACEME` call, which triggers a signal that we wait for.